tests/test_8schools.py

```diff
@@ -18,9 +21,51 @@ def schools_pm4():
     return obs
 
 
+def test_model_logp():
+    """
+    Recreate this with
+
+    ```python
+    import pymc3 as pm
+    import numpy as np
+
+    J = 8
+    y = np.array([28, 8, -3, 7, -1, 1, 18, 12])
+    sigma = np.array([15, 10, 16, 11, 9, 11, 10, 18])
+
+
+    with pm.Model() as eight_schools:
+        eta = pm.Normal("eta", 0, 1, shape=J)
+        mu = pm.Normal("mu", 1, 10)
+        tau = pm.HalfNormal("tau", 2.0)
+
+        theta = mu + tau * eta
+
+        pm.Normal("obs", theta, sigma, observed=y)
+
+    print(eight_schools.logp({'eta': np.zeros(8), 'mu': 0, 'tau_log__': 1}).astype(np.float32))
```

Comment:
tau_log__ --> __log_tau according to below

Reply:
`tau_log__` is from PyMC3! We spell it `__log_tau` now! :grin: